chronicle/Models/IOSDevice.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  IOSDevice.swift
+//  IOSDevice
+//
+//  Created by Alfonce Nzioka on 8/11/21.
+//
+
+import Foundation
+
+// information about the device
+// ref: https://developer.apple.com/documentation/uikit/uidevice
+struct IOSDevice :Codable {
+    var model: String
+    var version: String
+    var deviceId: String?
+    var name: String
+    var systemName: String
+    var localizedModel: String
+    
+    /// fully qualified java class name required when deserializing object
+    var className = "com.openlattice.chronicle.sources.IOSDevice"
+    
+    init(model: String, localizedModel: String, version: String, deviceId: String?, name: String, systemName: String) {
+        self.model = model
+        self.localizedModel = localizedModel
```

Comment:
What is `localizedModel`? Do we need it?

Reply:
https://developer.apple.com/documentation/uikit/uidevice/1620029-localizedmodel

used here: 
https://github.com/openlattice/chronicle-api/blob/main/src/main/java/com/openlattice/chronicle/sources/IOSDevice.kt